stock_whole_kit_constraint/models/stock_move.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    allow_partial_kit_delivery = fields.Boolean(
+        compute="_compute_allow_partial_kit_delivery",
+        compute_sudo=True,
+        default=True,
+    )
+
+    @api.depends(
+        "product_id.product_tmpl_id.allow_partial_kit_delivery", "state"
+    )
+    def _compute_allow_partial_kit_delivery(self):
+        """Take it from the product only if it's a kit"""
+        for move in self.filtered(
+            lambda x: x.product_id and x.state not in ["done", "cancel"]
```

Comment:
Watching to v13, we shouldn't do this for not having missing cache entries, but set the value for all. You can do it in advance.

Reply:
Ok changed and removed the weird default